charts/incubator/alfresco-dbp/values.yaml

```diff
@@ -51,4 +57,4 @@ alfresco-content-services:
         memory: "1500Mi"
 
 alfresco-sync-service:
-  enabled: true
\ No newline at end of file
+  enabled: false
```

Comment:
Does the sync service itself fail if the AMPs aren't applied?  If not, then perhaps we should leave it enabled just to continue to exercise as many of the components as possible.

Reply:
Yes, I’m pretty sure the pod failed to start or there were errors in the log which is what led me to disabling it but I’ll double check tomorrow.

Reply:
As I thought, an error during startup of the sync service:

```ERROR [2018-03-27 21:31:10.830] [main] org.alfresco.service.subscription.impl.config.ConfigurationImpl - Problem getting sync service configuration from repository: HTTP/1.1 404```

Sync Service needs the AMP installed as it provides webscripts it's expecting to find so disabling the sync service is correct unfortunately.